tests/js/spec/views/performance/transactionEvents.spec.jsx

```diff
@@ -47,6 +47,11 @@ describe('Performance > TransactionSummary', function () {
       url: '/organizations/org-slug/sdk-updates/',
       body: [],
     });
+    MockApiClient.addMockResponse({
+      method: 'GET',
+      url: `/organizations/org-slug/legacy-key-transactions-count/`,
+      body: [],
+    });
```

Comment:
This shouldn't be necessary on this page since you don't have key transactions.

Reply:
Good point! Removed!